src/material/datepicker/month-view.ts

```diff
@@ -180,7 +180,7 @@ export class MatMonthView<D> implements AfterContentInit, OnChanges, OnDestroy {
   /** The names of the weekdays. */
   _weekdays: {long: string, narrow: string}[];
 
-  constructor(private _changeDetectorRef: ChangeDetectorRef,
+  constructor(readonly _changeDetectorRef: ChangeDetectorRef,
```

Comment:
The change is fine, but what's the context behind it? Which other components need to access it?

Reply:
There's a Google internal datepicker spec based off this one. It's needed as part of implementing that.